app/views/activities/_filters.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with url: project_activities_path(current_project.id), method: :get, remote: true, class: 'd-flex flex-xl-row flex-column gap-3', data: { behavior: 'activity-filters' } do |f| %>
+  <div class="d-flex flex-column flex-sm-row w-100 gap-3">
+    <div class="w-100">
+      <%= f.label :user, class: 'form-label' %>
+      <%= f.select :user, options_from_collection_for_select(User.all, :id, :name, params[:user]), { include_blank: 'All users' }, { class: 'form-select' } %>
+    </div>
+    <div class="w-100">
+      <%= f.label :type, class: 'form-label' %>
+      <%= f.select :type, options_from_collection_for_select(Activity.select(:trackbable_type).distinct.pluck(:trackable_type), :to_s, :demodulize ), { include_blank: 'All types', selected: params[:type] }, { class: 'form-select' } %>
```

Comment:
```suggestion
      <%= f.select :type, options_for_select(Activity.pluck(:trackable_type).uniq), { include_blank: 'All types', selected: params[:type] }, { class: 'form-select' } %>
```

Reply:
That is how i had done it initially. However, there are cases that the types are namespaced within modules and they look bad on the dropdown. That way we benefit from the demodulize method for display reasons.

Reply:
That's fine, `options_for_collection_from_select` works too

The main changes here are the `:trackable_type` typo fix and the query adjustment. `Activity.pluck(:trackable_type).uniq` 

<img width="1537" alt="Screen Shot 2023-12-12 at 3 23 07 PM" src="https://github.com/dradis/dradis-ce/assets/78065527/062bb076-d803-4e06-86a4-032cfe55b4b1">